cnf-certification-test/platform/nodetainted/nodetainted.go

```diff
@@ -29,22 +28,35 @@ import (
 
 // NodeTainted holds information about tainted nodes.
 type NodeTainted struct {
-	timeout      time.Duration
-	ClientHolder *clientsholder.ClientsHolder
-	OCPContext   clientsholder.Context
+	ClientHolder      *clientsholder.ClientsHolder
+	OCPContext        clientsholder.Context
+	unitTestingResult bool
 }
 
-// NewNodeTainted creates a new NodeTainted tnf.Test.
-func NewNodeTaintedTester(timeout time.Duration, client *clientsholder.ClientsHolder, ctx clientsholder.Context) *NodeTainted {
+//go:generate moq -out nodetainted_moq.go . TaintedFuncs
+type TaintedFuncs interface {
+	runCommand(cmd string, ctx clientsholder.Context) (string, error)
```

Comment:
I believe it would make more sense to have the context as the first argument in all the functions and keep it as that in the rest of the code

Reply:
Done!